docs/GetStarted/getting_started_android_cmake.md

```diff
@@ -207,6 +207,51 @@ EXEC @abs
 i32=5
 ```
 
+### Dylib LLVM AOT backend
```

Comment:
Wrong heading nesting: move this to the bottom, below the Vulkan `#### Common issues` and ``##### Vulkan function `vkCreateInstance` not available`` sections